js/core.js

```diff
@@ -619,11 +626,73 @@ class ExtensionResources {
     }
 }
 
+class ContextMenu {
+    static async init() {
+        ContextMenu.updateContextMenu();
+        chrome.contextMenus.onClicked.addListener(ContextMenu.genericOnClick);
+        chrome.storage.onChanged.addListener(ContextMenu.updateContextMenu);
```

Comment:
Unsure if this is how I want to listen for option changes. Also, it doesn't seem to work well anyway...